client/Cargo.toml

```diff
@@ -4,25 +4,25 @@ version = "0.13.0"
 authors = [
     "Steven Roose <steven@stevenroose.org>",
     "Jean Pierre Dudey <jeandudey@hotmail.com>",
-    "Dawid Ciężarkiewicz <dpc@dpc.pw>"
+    "Dawid Ciężarkiewicz <dpc@dpc.pw>",
 ]
 license = "CC0-1.0"
 homepage = "https://github.com/rust-bitcoin/rust-bitcoincore-rpc/"
 repository = "https://github.com/rust-bitcoin/rust-bitcoincore-rpc/"
 description = "RPC client library for the Bitcoin Core JSON-RPC API."
-keywords = [ "crypto", "bitcoin", "bitcoin-core", "rpc" ]
+keywords = ["crypto", "bitcoin", "bitcoin-core", "rpc"]
 readme = "README.md"
 
 [lib]
 name = "bitcoincore_rpc"
 path = "src/lib.rs"
 
 [dependencies]
-bitcoincore-rpc-json = { version = "0.13.0", path = "../json"}
+bitcoincore-rpc-json = { version = "0.13.0", path = "../json" }
 
 log = "0.4.5"
-jsonrpc = "0.11"
+jsonrpc = "0.12.0"
 
 # Used for deserialization of JSON.
 serde = "1"
-serde_json = "1"
+serde_json = { version = "1", features = ["raw_value"] }
```

Comment:
I think we can just use the re-expected `serde_json` from `jsonrpc` now and save us maintaining the separate dep?

Reply:
Yeah that's right, the raw_value stuff doesn't work until a later version of serde_json anyway so this is cruft.

Reply:
resolved by 9d18340